node/src/daemon/setup_reporter.rs

```diff
@@ -373,14 +383,17 @@ impl SetupReporterReal {
         }
     }
 
-    fn make_command_line(setup: &SetupCluster) -> Vec<String> {
+    fn make_command_line(setup: &SetupCluster, blanked_out_params: Vec<String>) -> Vec<String> {
```

Comment:
I think I can see as far as where your thoughts were (I'd done something similar too and discarded it in the next step). Given this is really unclear code in overall I'm happy not to have to dive deeper and reason about why this probably isn't the best way and that's so because I've found that your implementation (or the only affected usage in production) lies with an empty vector passed to the second argument (see line 313 or the function of this name in `calculate_configured_setup` respectively).

That means whatever you did that appeased the test down in this file testing if we can zero out the `mapping-protocol` parameter it is not caused by the code change here since this should have no effect at all.

Please revert the changes in this function (signature + the iterator adding blanking out arguments back in).

Reply:
Good point.